### DISCUSSION TITLE
Welcome to makesmatheasy.github.io Discussions!

### DISCUSSION THREAD

**Discussion by rajinderpalsingh2001** [makesmatheasy/makesmatheasy#148] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vmk8826** [comment]
where can I get the link to the discord channel?

**Reply by rajinderpalsingh2001** [reply]
Are you a GSSOC'21 Participant?

**Reply by vmk8826** [reply]
yes I am

**Reply by rajinderpalsingh2001** [reply]
I have provided a link into the Comment section

https://github.com/makesmatheasy/makesmatheasy/issues/3076
